src/components/capture/creation/Store.ts

```diff
@@ -4,15 +4,6 @@ import { isEqual } from 'lodash';
 import create from 'zustand';
 import { devtools } from 'zustand/middleware';
 
-enum EventNames {
```

Comment:
It should be noted that each word in these string constants are capitalized. When this enumeration was broken down, only the first word in each string is capitalized. While a minor change, this is an opportunity to achieve consistency in the codebase.